teuthology/task/install.py

```diff
@@ -926,6 +929,105 @@ def _upgrade_deb_packages(ctx, config, remote, debs):
         ] + ['%s=%s' % (d, version) for d in debs],
     )
 
+@contextlib.contextmanager
+def rh_install(ctx,config):
```

Comment:
In any function call or signature, please separate arguments with spaces:
``def rh_install(ctx, config)``